### DISCUSSION TITLE
[Announcement] Homework 5 is out

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs348-information-security#452] [Announcements]
Hi all,
this is an announcement about homework 5.

Homework 5 is a programming assignment requiring you to exploit target programs that have buffer-overflow and integer-overflow.

Here is the GitHub classroom link for this homework:
(https://classroom.github.com/a/TZGASljr)

## Due date

Due: 05. 22 (Mon.) 23:59:59
Late Due: 05. 24 (Wed.) 23:59:59 (late submission penalty).

**Comment by m-spitfire** [comment]
There won't be gradescope for this homework? Only pushing to github is required, right?

**Comment by KAIST-JongchanPark** [comment]
@m-spitfire Oh, please submit your assignment into gradescope too!

**Comment by KihongHeo** [comment]
Write the documents `problemN.txt` in Korean if you are a native Korean speaker, in English otherwise.

If you have already submitted them in English, no worries. Just leave them.
